src/data/polynomial/algebra_map.lean

```diff
@@ -251,6 +251,15 @@ lemma coeff_zero_eq_aeval_zero' (p : R[X]) :
   algebra_map R A (p.coeff 0) = aeval (0 : A) p :=
 by simp [aeval_def]
 
+lemma aeval_commuting_diagram {S T U : Type*} [comm_ring S] [comm_ring T] [comm_ring U]
```

Comment:
I like the name, but I am afraid that mathlib convention is to call it something like `map_aeval_eq_aeval_map` or whatever. Also, the fact that you use `algebra R S` and `algebra T U` but `φ : R →+* T` and `ψ : S →+* U` (instead of some `alg_hom`) looks a little bit asymmetric, but I guess this is the form you need.

Reply:
Sure, I'll change that!